### PR TITLE
ADA-126 fix bug

### PR DESCRIPTION
**Pull request by SnobbyDragon**
no caps conventions bc those arent real
try catch to silence student dup error

### CODE REVIEW

File: acmw-db-app/components/AddAccountForm.jsx [hunk 1]
```diff
@@ -54,21 +57,29 @@ const AddAccountForm = (props) => {
...
           };
-          const res1 = await fetch('/api/student/create', requestOptionsStudent);
-          const result1 = await res1.json(); //returns the id
-          id = result1[0]["id"];
+          try {
+            const res1 = await fetch('/api/student/create', requestOptionsStudent);
+            const result1 = await res1.json(); //returns the id
+            id = result1[0]["id"];
+          } catch (err) {
+            // duplicate; selects student instead
```
Comment (jgeorge37) [inline acmw-db-app/components/AddAccountForm.jsx:74]: how do you know that it's a duplicate?

File: acmw-db-app/components/AddAccountForm.jsx [hunk 2]
```diff
@@ -54,21 +57,33 @@ const AddAccountForm = (props) => {
...
           };
-          const res1 = await fetch('/api/student/create', requestOptionsStudent);
-          const result1 = await res1.json(); //returns the id
-          id = result1[0]["id"];
+          try {
+            const res1 = await fetch('/api/student/create', requestOptionsStudent);
+            const result1 = await res1.json(); //returns the id
+            id = result1[0]["id"];
+          } catch (err) {
+            // hopefully duplicate; selects student instead
```
Comment (jgeorge37) [inline acmw-db-app/components/AddAccountForm.jsx:74]: this should use the error code that corresponds to attempted duplication, not another query

**Review by jgeorge37 — CHANGES_REQUESTED** [review]

**Comment by SnobbyDragon** [comment]
only resolved conflicts, not really merging everything from dev into this branch

**Comment by jgeorge37** [comment]
resolving conflicts always merges the whole branch

**Comment by SnobbyDragon** [comment]
> resolving conflicts always merges the whole branch

rats

**Review by jgeorge37 — APPROVED** [review]

**Pull request merged**